### PR TITLE
J.thebeau review 060 test development.md

### PR DESCRIPTION
**Pull request by jeremi-thebeau**
English review of 060 test development.

### CODE REVIEW

File: content/en/load-testing/Manual/060-test-development.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ description: >
 
 ### The Environment: your Favorite IDE
 
-XLT test cases are usually developed in Java, as JUnit test cases. This means you can work locally, in your IDE of choice. You will probably start out with one of our [sample test suites](../../test-suites), which come as Maven projects ready to import to your IDE of choice, build and run, and adjust those test cases to your needs, probably also adding new test cases. If you need to do some functional testing, test data may be added.
+XLT test cases are usually developed in Java, as JUnit test cases. This means you can work locally, in your IDE of choice. You will probably start out with one of our [sample test suites](../../test-suites), which come pre-configured as Maven projects ready to imported, built and run in your IDE of choice. You'll then want to adjust some test cases to your needs, and probably also add new ones. If you need to do some functional testing, test data may be added.
```
Comment (js-xc) [inline content/en/load-testing/Manual/060-test-development.md:15]: "ready to **be** imported..." or "ready to import, build..."? This seems not quite right...

File: content/en/load-testing/Manual/060-test-development.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ This is just an example and a different breakdown of flows and actions might mak
 
 ### Local Testing and Test Consolidation
 
-XLT tests are basically JUnit tests, so they can be run (and debugged) from the IDE, and you should do so repeatedly for consolidating the tests you wrote, because you really want stable and predictable tests for your load test. 
+XLT tests are basically JUnit tests; they can be run (and debugged) from the IDE. This should be done repeatedly while consolidation the various tests you have writen until a high level of stability is reached. Load testing necessitates stable and predictable test runs. Otherwise you will keep running into funtional test error and this can ultimately give you a false picture of your systems load capabilities.
```
Comment (js-xc) [inline content/en/load-testing/Manual/060-test-development.md:63]: during consolidation/while consolidating? Does this really work as it is?

also, "your system's load capabilities"?

File: content/en/load-testing/Manual/060-test-development.md [hunk 3]
```diff
@@ -96,13 +96,12 @@ A performance test case is also a regression test case - and if you want to [int
...
-* **Clean up at the end of your test case.** For example, after adding products to cart or addresses to address book make sure that the test case itself does remove them at the end. make sure each test execution starts under identical conditions. Validations may fail for the second test run if the previous one did not clean up all artifacts.
-* **Be prepared if the previous test run did not clean up and add some randomization where needed.** Test cases may fail because they try to add something that is already existing. The test case might add something during a failed test run that has not been finished (and therefore the clean up at the end of the test case has not been performed). Try to access objects or list entries randomly, so even in case the clean up did not work properly, the next execution has at least a chance to pass.
+* **Do not overload the test cases.** Focus on the (one) key purpose of the test case. It is better to create a lot of short tests than only a few large test cases That cover many features. This helps to maintain and debug the tests and it is also much easier to locate the root cause of real problems if a test case fails.
```
Comment (js-xc) [inline content/en/load-testing/Manual/060-test-development.md:99]: **t**hat cover (lowercase)

File: content/en/load-testing/Manual/060-test-development.md [hunk 4]
```diff
@@ -96,13 +96,12 @@ A performance test case is also a regression test case - and if you want to [int
...
+* **Do not test the same feature again and again.** If possible, do not test the same feature again and again in several test cases. Instead find the shortest way to get to a specific page! For example when testing different shopping bag or checkout features in several test cases, try to avoid searching for a product and browsing to the product detail page. Instead, start the test case by opening the product detail page directly. That way, if something happens to the search functionality, only the test cases covering search will fail. If search is used in all the test cases, then all of them will fail and the issue will be more difficult to debug.
+* **Use flows to perform repeating sequences.** If you realize that you're using an identical sequence of several calls more then once, then think about extracting it to a flow. Not only does it save you time when creating the test cases but also makes maintenance much easier. Adjusting an xpath will only have to be done once in the flow instead of repeating fixes in each single test case. For example loggin in or adding products to cart are typical sequences that have to be done in many test cases and are worth creating a flow.
```
Comment (js-xc) [inline content/en/load-testing/Manual/060-test-development.md:101]: loggin**g** in

Comment (jeremi-thebeau) [reply content/en/load-testing/Manual/060-test-development.md]: Fixed.

File: content/en/load-testing/Manual/060-test-development.md (hunk 3 above)
Comment (jeremi-thebeau) [reply content/en/load-testing/Manual/060-test-development.md]: Fixed.

File: content/en/load-testing/Manual/060-test-development.md (hunk 2 above)
Comment (jeremi-thebeau) [reply content/en/load-testing/Manual/060-test-development.md]: thanks, it does not.

Also, correct, thanks. Fixed.

File: content/en/load-testing/Manual/060-test-development.md (hunk 1 above)
Comment (jeremi-thebeau) [reply content/en/load-testing/Manual/060-test-development.md]: Fixed.

File: content/en/load-testing/Manual/060-test-development.md [hunk 5]
```diff
@@ -60,9 +60,9 @@ This is just an example and a different breakdown of flows and actions might mak
 
 ### Local Testing and Test Consolidation
 
-XLT tests are basically JUnit tests, so they can be run (and debugged) from the IDE, and you should do so repeatedly for consolidating the tests you wrote, because you really want stable and predictable tests for your load test. 
+XLT tests are basically JUnit tests; they can be run (and debugged) from the IDE. This should be done repeatedly while consolidating the various tests you have writen until a high level of stability is reached. Load testing necessitates stable and predictable test runs. Otherwise you will keep running into funtional test error and this can ultimately give you a false picture of your system's load capabilities.
```
Comment (js-xc) [inline content/en/load-testing/Manual/060-test-development.md:63]: have writ**t**en (sorry I missed this one before)

Comment (jeremi-thebeau) [reply content/en/load-testing/Manual/060-test-development.md]: Fixed.

**Pull request merged**